Path-Check/covid-safe-paths
# [1.0.x master] Bug/fix android crash permissions

#### Description:
`react-native-notifications` requires firebase integration for Android if you're configuring the notifications with permissions set to true. This [work-around](https://github.com/zo0r/react-native-push-notification#usage) comes from their docs, to avoid crashing Android when you're not using firebase.
#### Linked issues:

https://pathcheck.atlassian.net/browse/SAF-234

#### Screenshots:
![image](https://user-images.githubusercontent.com/2692908/81878522-5096af00-9556-11ea-8d19-bad48ef950ff.png)


#### How to test:
Run the app on Android.
Visit the Dashboard screen.
Disable & RE-Enable location in the Dashboard screen.
When I re-enabled location, I got a black screen with a Firebase error.
Expect that the app does not crash with the same red box error as shown above
Regression test on iOS
